tradeexecutor/strategy/pricing_model.py

```diff
@@ -40,11 +40,11 @@ class TradePricing:
     #: How much liquidity provider fees we are going to pay on this trade.
     #:
     #: Set to None if data is not available.
-    lp_fee: Optional[USDollarAmount] = None
+    lp_fee: Optional[list[USDollarAmount]] = None
 
     #: What was the LP fee % used as the base of the calculations.
     #:
-    pair_fee: Optional[BPS] = None
+    pair_fee: Optional[list[BPS]] = None
```

Comment:
There is no information on what these pairs are, see we need to add that to the information as well.

For typing hints `List` must be capitalised.

Please add `get_total_lp_fees()` helper that gets the sum of all fees.

Please cover `get_total_lp_fees()` in tests and documentation.